modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -60,12 +62,13 @@ Before you implement multiple client providers in Anypoint Platform, carefully r
 * When you change the client provider for an existing API instance, ensure that the API:
 ** Has no contracts
 ** Has no client provider-related policies applied, such as PingFederate and OpenID policies
-* When you promote and import an API:
-
-* If previously did not assign a client provider to an environment, after you assign an external client provider, all existing APIs in that environment still use the native Anypoint Platform client provider. 
+* If you previously did not assign an external client provider to an environment when promoting and importing an API, the native provider is used by default. 
 +
 You can then reassign the appropriate external client provider for that API.
+* If you assign an external client provider to an environment that was previously using a native provider, all existing APIs in that environment still use the native Anypoint Platform client provider. The new external IdP is used only by new APIs.
```

Comment:
This might break up a complicated sentence:
```suggestion
* If you assign an external client provider to an environment that was previously using a native provider: +
* Existing APIs in that environment continue to use the native Anypoint Platform client provider. 
* New API use the new external IdP.
```
OR
```suggestion
* If you assign an external client provider to an environment that was previously using a native provider, only new external APIs use that external IdP.
```

Reply:
Done! Thanks!